packages/sources/svelte/src/components/structure/VtmnAccordion/VtmnAccordion.svelte

```diff
@@ -0,0 +1,40 @@
+<script>
+  import { cn } from '../../../utils/classnames';
+  /**
+   * @type {boolean} Display left icon on summary.
+   */
+  export let withIconLeft = false;
+  /**
+   * @type {string} Summary of the accordion.
+   */
+  export let summary;
+  /**
+   * @type {boolean} Disable the accordion.
+   */
+  export let disabled = false;
+  /**
+   * @type {boolean} Open the accordion.
+   */
+  export let open = false;
+  let className;
+  /**
+   * @type {string} Custom classes to apply to the component.
+   */
+  export { className as class };
+  $: componentClass = cn(
+    'vtmn-accordion',
+    withIconLeft ? 'vtmn-accordion--with-icon-left' : '',
+    className,
+  );
+</script>
+
+<details class={componentClass} aria-disabled={disabled} {open}>
```

Comment:
I think you don't need to specify "open" prop as it's an HTML default property.
But if we want to set it, you will need to spread `$$restProps` to the `<details>` HTML block.